wiki/Article_Style_Guide/en.md

```diff
@@ -255,7 +299,7 @@ You may use the old game mode names (e.g. `Catch the Beat` or `Taiko`) only when
 
 ### Game Modifiers
 
-Game modifiers **must** be capitalised.
+Game modifiers **must** capitalise the first letter of each word.
```

Comment:
You usually don't capitalise articles and proportions in title case. Considering there are none of those in mod names(?) I'd say it's safe...